app/views/saml_idp/shared/saml_post_binding.html.erb

```diff
@@ -18,16 +20,17 @@
             <%= t('.no_js') %>
           </p>
 
-          <%= form_tag action_url, id: 'saml-post-binding' do %>
+          <%= simple_form_for('', url: action_url) do |f| %>
             <%= hidden_field_tag('csp_uris', csp_uris) if Rails.env.test? %>
             <%= hidden_field_tag(type, message) %>
             <% if params.key?(:RelayState) %>
               <%= hidden_field_tag('RelayState', params[:RelayState]) %>
             <% end %>
```

Comment:
could we reuse `saml_post_form.html.erb` here and pass in `:RelayState` as `form_params`?

Reply:
> could we reuse `saml_post_form.html.erb` here and pass in `:RelayState` as `form_params`?

Maybe! I can check

Reply:
There was some prior discussion about this approach with a general leaning toward the duplication? Not sure I'd totally agree, but might save this for a separate effort.

https://github.com/18F/identity-idp/pull/5624/files#r753741166